pylzy/lzy/api/v1/snapshot.py

```diff
@@ -106,21 +106,21 @@ def create_entry(self, name: str, typ: Type) -> SnapshotEntry:
         _LOG.debug(f"Created entry {e}")
         return e
 
-    async def get_data(self, entry_id: str) -> Result[Any]:
+    async def get_data(self, entry_id: str) -> Either[Any, Exception]:
         _LOG.debug(f"Getting data for entry {entry_id}")
         entry = self.__entry_id_to_entry.get(entry_id, None)
         if entry is None:
             raise ValueError(f"Entry with id={entry_id} does not exist")
 
         try:
             storage_uri = entry.storage_uri
-        except ValueError:
-            _LOG.debug(f"Error while getting data for entry {entry_id}")
-            return Nothing()
+        except ValueError as e:
+            _LOG.debug(f"Error while getting data for entry {entry_id}: {str(e)}")
```

Comment:
Do not use f-strings in logging, only good old `%s` formatting

Reply:
> Do not use f-strings in logging, only good old `%s` formatting

Why?

Reply:
Because it gives u more flexebility while logging setup and tweak and it is more optimised.
Also, 
https://docs.python.org/3/howto/logging-cookbook.html#formatting-styles 
https://docs.python.org/3/howto/logging.html#optimization